### DISCUSSION TITLE
Some suggestions

### DISCUSSION THREAD

**Discussion by nexus226** [ntsc-rs/ntsc-rs#87] [Ideas]
- Remember last window size/position (for standalone app)
- Select a default folder for presets and saving videos
- Option to toggle anti-aliasing for the zoom feature

**Comment by Degamisu** [comment]
Not sure about the antialiasing part, do you mean Bilinear Filtering? This is the blurry effect you get when zooming in on images (if supported by the viewer) antialiasing would smooth out edges of lines, but i dont see a use, since the input is most likely antialiased.

**Reply by valadaptive** [reply]
I think they're referring to the zoom filter, yeah. I think it'd be best to use nearest-neighbor at 2x scale or above, but linear below that. I just need to figure out how to do that in the UI framework I'm using (I've tried the obvious way and it doesn't work)

**Reply by nexus226** [reply]
![image](https://github.com/valadaptive/ntsc-rs/assets/105115185/8d593726-ccf1-464d-a8f9-f069ae9978c6)
When I use the zoom function included on ntsc-rs to get a closer look at my footage, it uses Nearest Neighbor, you can see by the squares in this screenshot. What I'm saying is to add a manual switch to use Bilinear Filtering or Nearest Neighbor when zooming in.

**Comment by max9830** [comment]
He/her should add the WEBAPP for people who cant use NTSCRS on bigger computers

**Reply by valadaptive** [reply]
Please don't make unrelated suggestions on existing posts.
